src/rpc/lz4_compressor.cc

```diff
@@ -71,7 +71,7 @@ class reusable_buffer {
         return _data.get();
     }
 
-    // Calls function fn passing to it a pointer to a temporary contigiuous max_size
+    // Calls function fn passing to it a pointer to a temporary contiguous max_size
```

Comment:
discriminating between `contiguous` and `continuous` is always hard for me, but I think this is right...